sources/packages/backend/apps/api/src/route-controllers/application-exception/_tests_/e2e/application-exception.aest.approveException.e2e-spec.ts

```diff
@@ -0,0 +1,221 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import { ApplicationExceptionStatus } from "@sims/sims-db";
+import { ClientTypeBaseRoute } from "../../../../types";
+import * as faker from "faker";
+import { createFakeApplicationWithApplicationException } from "../application-exception-helper";
+import { TestingModule } from "@nestjs/testing";
+import { ZBClient } from "zeebe-node";
+
+describe(`${ClientTypeBaseRoute.AEST}-ApplicationExceptionAESTController(e2e)-approveException`, () => {
```

Comment:
`${ClientTypeBaseRoute.AEST}` was added by mistake on a previous PR. The name of the controller is already specific to the client `ApplicationException*AEST*Controller`